README.md

```diff
@@ -344,15 +344,9 @@ Requirements: Linux or macOS with installed Docker.
 | 🔛 [Run Acra Load Balancing Example](https://github.com/cossacklabs/acra-engineering-demo/#example-8-load-balancing-with-acra) 🔛 |
 |---|
 
-## GDPR, HIPAA, CCPA
+## GDPR and HIPAA
```

Comment:
Looks like a merge conflict. The version in master is the correct one, @karenswry has recently expanded this section.

Reply:
Actually now I copied README.md from acra/master and wasn't changed - https://raw.githubusercontent.com/cossacklabs/acra/master/README.md
Now master has the same version as in PR.

Reply:
Here’s what I see at **master**’s tip:

- 2b88682b775e171e218088e99725d349e35df443 ("Added CCPA mention to README.md")

And here’s the commit I see as a tip of your branch **lagovas/ee-updates**:

- 38220692fe92c873377a218f342152c10f4ea307 ("add tests for query parsers")

```diff
$ git diff 2b88682b775e171e218088e99725d349e35df443..38220692fe92c873377a218f342152c10f4ea307 -- README.md | head
diff --git a/README.md b/README.md
index a7ecf8c..b03506a 100644
--- a/README.md
+++ b/README.md
@@ -344,15 +344,9 @@ Requirements: Linux or macOS with installed Docker.
 | 🔛 [Run Acra Load Balancing Example](https://github.com/cossacklabs/acra-engineering-demo/#example-8-load-balancing-with-acra) 🔛 |
 |---|
 
-## GDPR, HIPAA, CCPA
+## GDPR and HIPAA
```

So it seems that your master branch has gone stale at some point, then some merges happened, and now git thinks that all conflicts have been resolved (in favor of older version).

Blame & bisect shows that this commit is the culprit:

- 1fd032522e9699a01638ca4bb4b73fc95f75057c ("megrge ce")

Well, I’ll just go ahead and push the commit with a fixup that syncs README with master.